wsmaster/che-core-api-agent/src/main/resources/agents/scripts/org.eclipse.che.ls.json.script.sh

```diff
@@ -21,7 +21,7 @@ LS_DIR=${CHE_DIR}/ls-json
 LS_LAUNCHER=${LS_DIR}/launch.sh
 
 LINUX_TYPE=$(cat /etc/os-release | grep ^ID= | tr '[:upper:]' '[:lower:]')
-LINUX_VERSION=$(cat /etc/os-release | grep ^VERSION=)
+LINUX_VERSION=$(cat /etc/os-release | grep ^VERSION.*=)
```

Comment:
on alpine I have now
$ echo $(cat /etc/os-release | grep ^VERSION.*=)
VERSION_ID=3.4.0

but on ubuntu I have several values now

echo $(cat /etc/os-release | grep ^VERSION=)
VERSION="16.04 LTS (Xenial Xerus)"

user@87e1b3d386f3:/projects$ echo $(cat /etc/os-release | grep ^VERSION.*=)
VERSION="16.04 LTS (Xenial Xerus)" VERSION_ID="16.04"

it doesn't break ?
or it should only check for VERSION_ID ?